website/content/en/docs/ansible/k8s-modules.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Development Tips
-weight: 12
+title: Getting started with the k8s Ansible modules
+linkTitle: K8S Ansible Modules
+weight: 3
 ---
```

Comment:
Development Tips do not make really clear what it is about. So, updated the linkTitle to be == the small version of the doc title.

Reply:
I considered this change when initially moving the content over, but this particular doc contains information not related to k8s-modules. "Development tips" was the best I could think of that covered whats in this doc. IMO we probably ought to break this up, but since that will be a larger effort, it should be planned in an issue and done separately.

Reply:
The name of the doc is `Getting started with the k8s Ansible modules` so I am not sure if we should change it for tips for example.

Reply:
I think thats just a problem with the number of ##s,  "Getting Started with the k8s Ansible modules" is a subsection of development tips.

Reply:
The problem is development tips do not clarify what is the doc about. Also, I am as a developer would looking for info over how to work with ks8 modules and I will NOT search for development tips. It can be anything. So, IMO for now, we should keep the **current name** of the docs and then, in the future see how we can improve it and organize better. is it make sense? WDYT?

Reply:
"Getting started with the k8s Ansible modules" is not the current title of this doc https://github.com/operator-framework/operator-sdk/blob/v0.16.x/doc/ansible/dev/developer_guide.md

Reply:
I catcher what do you mean. It is reverted. You are right.